whatsagent/api/views.py

```diff
@@ -0,0 +1,143 @@
+import typing
+import json
+
+import requests
+from django.http.request import HttpRequest
+from django.http.response import JsonResponse
+from django.http.response import HttpResponse
+from django.views.decorators.http import require_http_methods
+
+from whatsagent.api.serializers.serializers import SignupSerializer
+
+from rest_framework import serializers
+from whatsagent.api.models import User
+
+# Utility to use gmail api
+import googleapiclient.errors
+from django.conf import settings
+import ssl
+import smtplib
+from email.message import EmailMessage as pymail
+from enum import Enum
+from rest_framework.decorators import api_view
+from rest_framework.exceptions import APIException
+from django.db.transaction import atomic
+
+
+class EmailStatus(Enum):
+    SUCCESS = 'email_sending_success'
+    FAILURE = 'email_sending_failure'
+    PROGRESS = 'email_sending_in_progress'
+
+
+class ServiceUnavailable(APIException):
+    status_code = 503
+    default_detail = 'Service Unavailable'
+    default_code = 'service_unavailable'
+
+
+@require_http_methods('GET')
+def get_sample(r: HttpRequest) -> HttpResponse:
```

Comment:
Remove `get_sample`, it was `get_sample` as it's not supposed to be used anywhere, it's just an example code.